### DISCUSSION TITLE
Tasmota 13.3.0.5 Berry set_flex_flow / set_flex_align not working

### DISCUSSION THREAD

**Discussion by xostia** [arendst/Tasmota#20697] [Q&A] [answered]
Hi,

We are working in a custom version of Tasmota to implement a dark sky photometer.

It has an small OLED screen with some information and menu. I have it working in 13.1 but now I tried to migrate to 13.3 and in the Berry console I get these kind of errors and the display is black:

BRY: failed to run compiled code 'attribute_error' - the 'lv_obj' object has no attribute 'set_flex_flow'
BRY: failed to run compiled code 'attribute_error' - the 'lv_obj' object has no attribute 'set_flex_align"

... It is part of the Berry code to show the information in the screen. It WORKS in Tasmota 13.1 but NOT in Tasmota 13.3

The screen itself is well configured because I tested some simple LVGL samples and there is no problem... but Berry crashes with those errors trying to create different elements in the screen, like this:

        var topBar = lv.lv_obj(self.bodyContainer)
        topBar.remove_style_all()
        topBar.set_flex_flow(lv.FLEX_FLOW_ROW)
        topBar.set_flex_align(lv.FLEX_ALIGN_SPACE_BETWEEN, lv.FLEX_ALIGN_CENTER, lv.FLEX_ALIGN_CENTER)
        topBar.set_size(160, 10)

Can anybody help me to solve this? is it a bug? some change in the Berry->C mapping?

Thank you.

**Comment by s-hadinger** [comment]
We've moved to LVGL 9 so there might be some breaking changes. But in this case it's just the new mapping missed the layouts. This is fixed in #20701

**Reply by xostia** [reply, marked as answer]
OK, perfect, thank you :-)
